commit 25bb992b800c30dadebbe7587a5f4e104aff2ff0 (refs/changes/86/386/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2016-08-30T10:35:19-07:00 (3 years, 1 month ago)
    
    OS-5638 drop superfluous proc mount in imgadm's chroot-gtar
    Reviewed by: Todd Whiteman <todd.whiteman@joyent.com>
    Reviewed by: Patrick Mooney <patrick.mooney@joyent.com>

diff --git a/src/img/sbin/chroot-gtar b/src/img/sbin/chroot-gtar
index cbebe37c..15fb1bc1 100755
--- a/src/img/sbin/chroot-gtar
+++ b/src/img/sbin/chroot-gtar
@@ -123,10 +123,8 @@ cp /usr/img/lib/nomknod.so $LIBDIR/
 mkdir -p $LIBDIR/64/
 cp /usr/img/lib/64/nomknod.so $LIBDIR/64/
 
-TO_UMOUNT="$TO_UMOUNT $ROOTDIR/proc $ROOTDIR/dev/fd"
-TO_REMOVE="$TO_REMOVE $ROOTDIR/proc $ROOTDIR/dev/fd"
-mkdir -p $ROOTDIR/proc
-mount -F proc proc $ROOTDIR/proc
+TO_UMOUNT="$TO_UMOUNT $ROOTDIR/dev/fd"
+TO_REMOVE="$TO_REMOVE $ROOTDIR/dev/fd"
 [[ ! -e $ROOTDIR/dev ]] && TO_REMOVE="$TO_REMOVE $ROOTDIR/dev"
 mkdir -p $ROOTDIR/dev/fd
 mount -F fd fd $ROOTDIR/dev/fd
